sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/implementation/logs/AzureLogAnalyticsImplBuilder.java

```diff
@@ -185,22 +190,35 @@ public AzureLogAnalyticsImplBuilder addPolicy(HttpPipelinePolicy customPolicy) {
         return this;
     }
 
+    /**
+     * Sets the audience to use for authentication with Azure Active Directory. The Azure Public Cloud audience will be
+     * used if the property is null.
+     * @param audience audience to use for authentication with Azure Active Directory. The Azure Public Cloud audience
+     * will be used if the property is null.
+     * @return the {@link AzureLogAnalyticsImplBuilder}.
+     */
+    public AzureLogAnalyticsImplBuilder audience(String audience) {
+        this.audience = audience;
+        return this;
+    }
```

Comment:
Does the autorest supports this directly? Or was this a manual edit?

Reply:
This was a manual edit. AutoRest doesn't support this yet but will soon be supported.